zebrad/src/components/mempool/tests.rs

```diff
@@ -0,0 +1,50 @@
+use super::*;
+use color_eyre::Report;
+use std::collections::HashSet;
+use storage::tests::unmined_transactions_in_blocks;
+use tower::ServiceExt;
+
+#[tokio::test]
+async fn mempool_service_basic() -> Result<(), Report> {
+    // Using the mainnet for now
+    let network = Network::Mainnet;
+
+    // get the transactions in block one from the Zcash blockchain.
+    let block_one_transactions = unmined_transactions_in_blocks(1, network);
+
+    // Start the mempool service
+    let mut service = Mempool::new(network);
+
+    // Insert the first transaction from block one into the mempool storage.
+    service
+        .storage
+        .insert(block_one_transactions.1[0].clone())?;
```

Comment:
This test looks like a good test, but it doesn't do what the comments and variable names say it does.
Blocks start at height 0, and there are only coinbase transactions for the first 100 blocks.

```suggestion
    // get the genesis block transactions from the Zcash blockchain.
    let genesis_transactions = unmined_transactions_in_blocks(0, network);

    // Start the mempool service
    let mut service = Mempool::new(network);

    // Insert the genesis block coinbase transaction into the mempool storage.
    service
        .storage
        .insert(genesis_transactions.1[0].clone())?;
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2651/commits/0577085d87787bf3d8961c10a3442d85edf01d9c